chajda/tsquery/augments.py

```diff
@@ -52,62 +52,185 @@ def augments_gensim(lang, word, config=Config(), n=5):
 
     return words
 
-
+import os
 import fasttext
 import fasttext.util
+from annoy import AnnoyIndex
+
+
+chajda_path = os.path.dirname(__file__)
```

Comment:
it's a good idea to break out this path into it's own top level variable, but the name could be better;  it should somehow include the word `data` so that we know that's where data is getting stored

Reply:
a comment would also be helpful